src/test/scala/it/pagopa/pdnd/interop/uservice/partymanagement/persistence/v1/ProtobufConversionSpecs.scala

```diff
@@ -0,0 +1,409 @@
+package it.pagopa.pdnd.interop.uservice.partymanagement.persistence.v1
```

Comment:
could it be better to test `def`s in the serializer package object?
Those are the functions actually used for conversions.
I say this because I see that some conversions does not use utils methods